src/utils/fund_utils.py

```diff
@@ -1,8 +1,46 @@
 import re
+from typing import Dict
+
+from .colors import RgbColor, HlsColor
+
+FUND_GROUP_BASE_COLORS: Dict[str, RgbColor] = {
+    "PASSIVE": RgbColor(red=200, green=40, blue=40),
+    'ACTIVE': RgbColor(red=200, green=40, blue=40),
+    'BALANCED': RgbColor(red=230, green=130, blue=0),
+    'FIXED_INCOME': RgbColor(red=80, green=140, blue=80),
+    'OTHER': RgbColor(red=100, green=100, blue=100)}
```

Comment:
I could see quite a lot of more groups in the json file. If already agreed with the customer then this is fine otherwise, they all fall into OTHER category and will have the same color if the risk is also the same.

Reply:
There are 4 official groups and all others are concidered as "others"